frontend/src/components/HeadSail.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {getHeadSailTrim} from "../service/TrimDataService";
+
+export default function HeadSail() {
+    return (
+        <div>
+            {getHeadSailTrim()}
```

Comment:
As `getHeadSailTrim` will, hopefully, return a promise you need to do the request with useEffect on component mount and set a state accordingly.